python/BioSimSpace/_SireWrappers/_molecule.py

```diff
@@ -1242,17 +1243,20 @@ def _toPertFile(self, filename="MORPH.pert", zero_dummy_dihedrals=False,
                values. This allows the user to refer to properties with their
                own naming scheme, e.g. { "charge" : "my-charge" }
 
+           pert_type : str
+               String that specifies whether to use the 'standard' or 'multistep'
+               approach in perturbing the LJ, bond and angle terms.
+
            Returns
            -------
 
            molecule : Sire.Mol.Molecule
                The molecule with properties corresponding to the lamda = 0 state.
         """
-
-        if not self._is_merged:
```

Comment:
FYI, there are several cases below this line with the same issue

Reply:
This is because you've changed the name of the `self` argument in the method signature to `molecule`. While `self` is purely convention, it is good to be consistent so that it is obvious that this refers to the object itself, rather than a molecule that was passed in. As such, could you revert things back to how they were.

Reply:
Ah! I must have changed this during development last winter as I was importing the `toPertFile()` directly. See 91b34392651e4f8b049e4ae844bc426afa595f23